internal/chart/post_render.go

```diff
@@ -27,51 +33,68 @@ func (p *postRender) Run(renderedManifests *bytes.Buffer) (modifiedManifests *by
 		return nil, err
 	}
 
-	fs := filesys.MakeFsInMemory()
-	if err := fs.Mkdir(p.namespace); err != nil {
-		return nil, err
-	}
-
-	var postrenderFound bool
+	finalBuffer := renderedManifests
 	for _, cm := range configMapList.Items {
-		if !strings.HasSuffix(cm.Name, "-postrender") {
+		fwPatch := strings.HasSuffix(cm.Name, "-postrender")
+		var appPatch bool
+		for _, dv := range p.deploymentVersions {
```

Comment:
A map looks like a better fit for `deploymentVersions`: 
1. no need to have this `range` loop
2. a map is a set, no order expectations and so on

Reply:
in that case we would need to split the string and check if there is some deployment version inside there. it's same in terms of perfomance, and we are keeping map with empty struct which could be confusing